easybuild/easyconfigs/l/LinBox/LinBox-1.7.0-gfbf-2022a.eb

```diff
@@ -24,14 +18,14 @@ checksums = ['6d2159fd395be0298362dd37f6c696676237bc8e2757341fbc46520e3b466bcc']
 dependencies = [
     ('FFLAS-FFPACK', '2.5.0'),
     ('FLINT', '2.9.0'),
-    ('Givaro', '4.2.0'),
```

Comment:
Based on your description of the PR, I don't understand this removal. I understood what you wrote as it was getting Givaro linking info from pkg-config, not that Givaro was no longer a thing at all? Skimming the source code for linbox I still see givaro includes.

Reply:
Seems that was a bit to brief: It gets fflas-ffpack from pkgconfig and that (pkgconf result) includes potential options for Givaro as well. So it doesn't search for givaro (anymore) at all.

FFLAS-FFPACK depends on Givaro: https://github.com/easybuilders/easybuild-easyconfigs/blob/3cdfc1af78d318d2071dd8bf489cdaf177a81ef8/easybuild/easyconfigs/f/FFLAS-FFPACK/FFLAS-FFPACK-2.5.0-gfbf-2022a.eb#L26

As we don't pass anything related to Givaro to this installation anymore but from the point of view of LinBox configure it only needs fflags-ffpack I removed the explicit dependency. So if in the future fflas-ffpack & LinBox don't depend on givaro anymore this is not missed.